search/find-place/src/main/java/com/esri/samples/find_place/FindPlaceController.java

```diff
@@ -82,7 +88,7 @@ public void initialize() {
     callout.setLeaderPosition(Callout.LeaderPosition.BOTTOM);
 
     // create a locatorTask task
```

Comment:
Just noticing this comment as well:
```suggestion
    // create a locator task
```